src/main/eo/org/eolang/txt/text.eo

```diff
@@ -102,3 +102,9 @@
   #  We should implement checking that string is alphabetic
   #  only via EOLANG code. We can do it by using reduce method.
   [] > is-alphabetic /bool
+
+  # Returns concatenaion of two strings
+  # @todo #44:30min Current implementation using Java.
+  #  We should implement concatenation of two strings
+  #  only via EOLANG code. We can do it by using reduce method.
+  [other] > concatted /string
```

Comment:
@EugeneDar we can implement it by eo fast enough using `text.joined`. I think it would be better to do it in this pull request instead of later via pdd.